subsys/bluetooth/audio/Kconfig.has

```
@@ -65,7 +65,10 @@ config BT_HAS_PRESET_COUNT
 	  This option sets the number of Hearing Access Service Presets
 	  that can be registered. Setting this value to 0 disables Presets support.
 
-if BT_HAS_PRESET_COUNT > 0
+config BT_HAS_PRESET_SUPPORT
+	def_bool BT_HAS_PRESET_COUNT > 0
```

The alternative here would be
```suggestion
	bool # hidden
	default y if BT_HAS_PRESET_COUNT > 0
```

Not sure which is preferred, but I guess either one works :)

---

def_bool is the one suggested on Zephyr website